styles/variables/_typography.scss

```
@@ -9,6 +9,7 @@ $font-size-m: 1.25rem; // 20px
 $font-size-l: 1.5rem; // 24px
 $font-size-xl: 2rem; // 32px
 $font-size-labels: 1rem; // 16px
+$font-size-xll: 2.25rem;
```

Typo :)
Also it should be moved a line up, so that the order makes a bit more sense.

```suggestion
$font-size-xxl: 2.25rem;
```

---

Done